receiver/splunkhecreceiver/README.md

```diff
@@ -43,7 +43,7 @@ The following settings are optional:
       Note: Both `key_file` and `cert_file` are required for TLS connection.
     * `key_file`: Specifies the key file to use for TLS connection. Note: Both
       `key_file` and `cert_file` are required for TLS connection.
-* `raw_path` (default = '/services/collector/raw'): The path accepting [raw HEC events](https://docs.splunk.com/Documentation/Splunk/8.2.2/Data/HECExamples#Example_3:_Send_raw_text_to_HEC). Only applies when the receiver is used for logs.
+* `raw_path` (default = '/services/collector/raw'): The path accepting [raw HEC events](https://help.splunk.com/en/splunk-enterprise/get-data-in/get-started-with-getting-data-in/10.0/get-data-with-http-event-collector/format-events-for-http-event-collector#example-5-curl-statement-sending-raw-data-0). Only applies when the receiver is used for logs.
 * `splitting` defines the splitting strategy used by the receiver when ingesting raw events. Can be set to "line" or "none". Default is "line".
 * `health_path` (default = '/services/collector/health'): The path reporting [health checks](https://docs.splunk.com/Documentation/Splunk/9.0.1/RESTREF/RESTinput#services.2Fcollector.2Fhealth).
```

Comment:
Looks like this line needs updating too, according to https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/17369386737/job/49302040771?pr=42385

Reply:
thanks updated